Source/Initialization/WarpXInitData.cpp

```diff
@@ -105,6 +105,183 @@ WarpX::PostProcessBaseGrids (BoxArray& ba0) const
     }
 }
 
+void
+WarpX::PrintMainPICparameters ()
+{
+    amrex::Print() << "-------------------------------------------------------------------------------\n";
+    amrex::Print() << "--------------------------- MAIN EM PIC PARAMETERS ----------------------------\n";
+    amrex::Print() << "-------------------------------------------------------------------------------\n";
+
+    // Print geometry dimensionality
+    amrex::ParmParse pp_geometry("geometry");
+    std::string dims;
+    pp_geometry.query( "dims", dims );
+    if (dims=="1") {
+      amrex::Print() << "Geometry:             | 1 (Z)" << "\n";
+    }
+    else if (dims=="2") {
+      amrex::Print() << "Geometry:             | 2 (XZ)" << "\n";
+    }
+    else if (dims=="3") {
+      amrex::Print() << "Geometry:             | 3 (XYZ)" << "\n";
+    }
+    else if (dims=="RZ") {
+      amrex::Print() << "Geometry:             | 2 (RZ)" << "\n";
+    }
```

Comment:
Now that I see it again, I think it will be clearer if we add "D" after the dimensionality number, too:
```suggestion
    if (dims=="1") {
      amrex::Print() << "Geometry:             | 1D (Z)" << "\n";
    }
    else if (dims=="2") {
      amrex::Print() << "Geometry:             | 2D (XZ)" << "\n";
    }
    else if (dims=="3") {
      amrex::Print() << "Geometry:             | 3D (XYZ)" << "\n";
    }
    else if (dims=="RZ") {
      amrex::Print() << "Geometry:             | 2D (RZ)" << "\n";
    }
```